docs/releases/3.2.txt

```diff
@@ -322,6 +322,9 @@ Management Commands
 * :djadmin:`loaddata` now supports fixtures stored in XZ archives (``.xz``) and
   LZMA archives (``.lzma``).
 
+* :djadmin:`dumpdata` now can save fixtures in ``gz``, ``bz2``, ``lzma``, or
+  ``xz`` format.
```

Comment:
Wedon't call the input/output files as 'fixtures' in the rest of the command documentation so I don't think we should do so here, also you missed `.zip`:

```suggestion
* :djadmin:`dumpdata` now can compress data in the ``gz``, ``bz2``, ``lzma``, ``zip``
  or ``xz`` formats.
```

Reply:
Thanks I'll update the words as you suggested.
As I wrote below zip archive format is not supported.